jwplatform/common.py

```diff
@@ -0,0 +1,9 @@
+import os
+
+
+def get_file_size(file):
```

Comment:
I have the same thoughts here as I do for `constants.py`. That said there are multiple ways to do this in a single line of python without writing a new function: https://stackoverflow.com/questions/2104080/how-can-i-check-file-size-in-python

Reply:
@emptyflash - The code was previously implemented the same way as Matt suggests, and it was changed because you wanted a "file type" object. So, I changed it to what it's currently. Let me know what you want it to be like finally... I prefer to keep things simple and hence had the very simple version in the first cut.

Reply:
Yeah I don't have strong feelings about either approach. I just typically tend to like APIs that accept a file over a filepath

Reply:
Reverted back to the simple form.